docs/Basics.md

```diff
@@ -153,40 +173,62 @@ When not provided ignore file contains the following entries:
 node_modules
 ```
 
-## CLI Configuration
+## Complex recipes
+
+Exoframe also provides support for third-party complex deployment recipes.  
+They allow to quickly and easily deploy complex projects.
+
+For example, you can deploy Wordpress with PHPMyAdmin by simply executing `exoframe setup` and entering [exoframe-recipe-wordpress](https://github.com/exoframejs/exoframe-recipe-wordpress) as desired recipe.
+
+You can find the list of available recipes [on npm](https://www.npmjs.com/search?q=exoframe-recipe).
+
+## Exoframe CLI - Commands
+
+| Command              | Description                                                          |
+| -------------------- | -------------------------------------------------------------------- |
+| deploy [path]        | Deploy specified path                                                |
+| config               | Generate or update project config for current path                   |
+| list                 | List currently deployed projects                                     |
+| rm <id>              | Remove existing deployment or project                                |
+| log <id>             | Get logs for existing deployment or project                          |
+| template [ls, rm]    | Add, list or remove deployment templates from the server             |
+| setup [recipe]       | Setup a complex recipe deployment                                    |
+| token [ls, rm]       | Generate, list or remove deployment tokens                           |
+| secret [new, ls, rm] | Create, list or remove deployment secrets                            |
+| login                | Login into Exoframe server                                           |
+| endpoint [url]       | Selects or adds the endpoint of Exoframe server                      |
+| rm-endpoint [url]    | Removes an existing endpoint of Exoframe server                      |
+| update [target]      | Gets current versions or updates given target (server, traefik, all) |
+| completion           | Generates bash completion script                                     |
+
+## Exoframe CLI - Special Commands
+
+Exoframe CLI has a number of special commands, specifically:
+
+- `exoframe logs exoframe-server` - will return current server logs (only works when running server as container)
+
+## Exoframe CLI - Configuration
 
 Exoframe stores its config in `~/.exoframe/cli.config.yml`.  
 Currently it contains list of endpoint URLs with associated usernames and authentication tokens:
 
 ```yaml
-endpoint: "http://localhost:8080" # your endpoint URL, defaults to localhost
+endpoint: 'http://localhost:8080' # your endpoint URL, defaults to localhost
```

Comment:
Please revert this. As I've mentioned earlier - I'm using prettier to format everything and it defaults to `"` in yaml.

Reply:
ops, prettier did this 😂, i think i changed the setting

Reply:
I should really add a prettierrc to the project 🤔